horizon: 16
speed: 1.0
steering_angle: 0.0
model_path: models/mobilenet_dummy.pt
